libpldmresponder/meson.build

```diff
@@ -80,9 +80,15 @@ if get_option('oem-ibm').enabled()
     '../oem/ibm/libpldmresponder/file_io_type_vpd.cpp',
     '../oem/ibm/host-bmc/host_lamp_test.cpp',
     '../oem/ibm/libpldmresponder/bios_oem_ibm.cpp',
+    '../oem/ibm/pfw-sms/pfw_sms_menu_ibm.c',
```

Comment:
Please change the .c extension to .cpp and .hpp. Also the folder structure is under ibm you may omit it from the file name itself.

Reply:
Okay, I will do this.